README.md

```diff
@@ -1,4 +1,47 @@
-# Jarivs Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
+
+# DEREK_SONG . Jarvis Consulting
+
+My name is Derek Song, and I am currently working with Jarvis as a data engineer. My role is to utilize Java and Scala programming languages to build applications. I am proficient in Java, Linux, Bash, SQL, RDBMS, JDBC and DAO design pattern. Before that, I have achieved a Master's Degree and Bachelor's Degree in Engineering, and I have strong programming fundamentals and learning skills. Besides, I worked in TS Tech Canada Inc. for three years, where I built very good communication skills and achieved hands-on experience of working with team and contractors. I really enjoyed my work with Jarvis, and I would love to dig much deeper into this area.
+
+## Skills
+**Proficient:** Java, Maven, Bash, SQL, Agile/Scrum, Docker, RDBMS, JDBC, DAO, Git/GitHub, Microservice, Postman
+
+**Competent:** Spring and SpringBoot framework, Oracle 
+
+**Familiar:**: Spark, Scala, Hive, Kafka, Nifi
+
+## Development Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_derek](https://github.com/jarviscanada/jarvis_data_eng_derek)
```

Comment:
please read the instruction carefully. you need to create the `feature/resume` branch from the master. Please recreate a feature/resume2 branch

git checkout master
git checkout -b feature/resume2
#update your resume me and pdf
#commit 
#master PR